recipes/r-treesim/meta.yaml

```diff
@@ -15,7 +15,6 @@ source:
 build:
   skip: True  # [win]
   number: 0
-  noarch: generic
```

Comment:
Any reason why that is removed?

Reply:
The linter complained (see above: *noarch packages can't have selectors. If the selectors are necessary, please remove noarch: generic* )

Reply:
Ah. The solution in that case is to remove `skip: True  # [win]`... as noarch recipes are build on linux after feedstock conversion ...

Reply:
They are build **only** on linux then? This could help as well for `r-geiger`?

Reply:
It would help to split the noarch recipes into their on PR, where then the non-linux builds can be ignored.

Reply:
I see! Will do so :-)